common/src/project.rs

```diff
@@ -48,7 +56,19 @@ impl ProjectName {
                 || byte == b'-'
         }
 
-        !(hostname.bytes().any(|byte| !is_valid_char(byte))
+        fn is_profanity_free_and_not_reserved(hostname: &str) -> bool {
+            static INSTANCE: OnceCell<HashSet<String>> = OnceCell::new();
+            INSTANCE.get_or_init(|| HashSet::from(["Shuttle.rs".to_string()]));
+
+            let censor = Censor::Standard
+                + Censor::Sex
+                + Censor::Custom(INSTANCE.get().expect("Reserved words not set").clone());
+            return !censor.check(hostname);
+        }
+
+        !(hostname
+            .bytes()
+            .any(|byte| !is_valid_char(byte) | !is_profanity_free_and_not_reserved(hostname))
             || hostname.ends_with('-')
```

Comment:
```suggestion
            .any(|byte| !is_valid_char(byte))
            || !is_profanity_free_and_not_reserved(hostname)
            || hostname.ends_with('-')
```
I think this is what @coszio means. That way the same `is_profanity_free_and_not_reserved(hostname)` is not made with every byte check

Reply:
Easy fix, I'll do it now! :)